src/Components/NavBar/Menus/DropDownMenu.tsx

```diff
@@ -0,0 +1,110 @@
+import { Box, color, Flex, Menu, MenuItem } from "@artsy/palette"
+import { AnalyticsSchema } from "Artsy"
+import { useTracking } from "Artsy/Analytics/useTracking"
+import React from "react"
+import styled from "styled-components"
+import { DropDownSection } from "./DropDownSection"
+
+interface DropDownNavMenuProps {
+  width?: string
+  paddingTop?: number
+  paddingBottom?: number
+  menu: any
+}
+
+export const DropDownNavMenu: React.FC<DropDownNavMenuProps> = ({
+  width = "100%",
+  menu,
+}) => {
+  const { trackEvent } = useTracking()
+
+  const trackClick = event => {
+    const link = event.target
+    const text = link.innerText
+    const href = link.parentNode.parentNode.getAttribute("href")
+
+    trackEvent({
+      action_type: AnalyticsSchema.ActionType.Click,
+      context_module: AnalyticsSchema.ContextModule.HeaderArtworksDropdown,
+      subject: text,
+      destination_path: href,
+    })
+  }
+
+  // const getSubMenu = text => {
+  //   menu.links.filter(item => {
+  //     return item.text === text
+  //   })[0]
+  // }
+
+  const mediumLinks = menu.links.filter(item => {
+    return item.text === "Medium"
+  })[0]
```

Comment:
If `filter` returns undefined, then `[0]` access could result in an error. Since this invocation is repeated a few times we could make things safer in the commented out helper function just above.

Reply:
In theory they should always return something because the data is hardcoded. However, that might not be the case forever. I could conditional render the `DropDownSection` component inside of `DropDownMenu` (i.e.):

```tsx
  {mediumLinks.length > 0 && <DropDownSection section={mediumLinks} />
```

or handle this inside of `DropDownSection` by returning `null` when the section prop is undefined.

Thoughts??

Reply:
If it's hardcoded I think its fine for now! We'd want this to blow up early in that case, vs mask an error (like missing data for a menu item). When its dynamic and controlled from elsewhere (like say MP) then it would be good to add guards since upstream issues could involve any number of things.